### ISSUE TITLE
feat: TTL for fuse table

### ISSUE THREAD

**Issue by BohuTANG** [datafuselabs/databend#7828]
**Summary**

Segment Level TTL to expire the data.

**Comment by caibirdme** [comment]
any update for this feature?

**Comment by BohuTANG** [comment]
@caibirdme Hi, no update yet.

Since Databend uses object storage, which is cheaper than local disk, space isn't an issue, and there is a vacuum command(the storage space will reclaim if data is deleted or a table is dropped):
https://docs.databend.com/sql/sql-commands/ddl/table/vacuum-table

Future plans to adjust this remain undecided.

**Comment by caibirdme** [comment]
in vacuum-table doc, there's a setting `DATA_RETENTION_TIME_IN_DAYS `, is it a global setting? How can I make this setting vary from table to table? And is this user's task to run vaccum table routinely?

**Comment by TCeason** [comment]
> in vacuum-table doc, there's a setting `DATA_RETENTION_TIME_IN_DAYS `, is it a global setting? How can I make this setting vary from table to table? And is this user's task to run vaccum table routinely?

Yes it's a global setting now. Maybe cc @lichuang can provide help

**Comment by BohuTANG** [comment]
> in vacuum-table doc, there's a setting `DATA_RETENTION_TIME_IN_DAYS `, is it a global setting? How can I make this setting vary from table to table? And is this user's task to run vaccum table routinely?

We are planning to add this option to table.
